dendrite-config.yaml

```diff
@@ -0,0 +1,73 @@
+# The config file version format
+version: v0
```

Comment:
Nit: What's the intent behind this? For Dendrite to whine if you're running a too new/old version of the config file? If so, it'd be nicer if this were an `int` rather than a `string`.